website/pages/five_safes_tes/quickstart/hello_world.mdx

```diff
@@ -0,0 +1,149 @@
+import { Cards, Tabs, Steps, Callout } from "nextra/components";
+
+# Send Hello World submission
+
+This page will guide you through sending your first "Hello World" submission to the Five Safes TES weave.
```

Comment:
This is a little more than this, worth saying it's also running 5ST?

Reply:
Also is this around the place we should set the expectation for how long this will take to happen? Like expected time 1 hour